storage-node/src/commands/server.ts

```diff
@@ -350,7 +350,11 @@ Supported values: warn, error, debug, info. Default:debug`,
         },
         x_host_id: X_HOST_ID,
       })
-      app.listen(port, () => logger.info(`Listening on http://localhost:${port}`))
+      const server = app.listen(port, () => logger.info(`Listening on http://localhost:${port}`))
+
+      // INFO: https://nodejs.org/dist/latest-v18.x/docs/api/http.html#serverrequesttimeout
+      // Set the server request timeout to 0 to disable it. This was default behaviour pre Node.js 18.x
+      server.requestTimeout = 0
```

Comment:
I think it makes sense to have the timeout disabled, provided the node is always behind a reverse proxy and not exposed on a public interface.

Wouldn't a large value of say 20min be better that the default of 5min?

Reply:
Or disabled timeout but make sure docs and our default deployment configs do not expose public interface.. ?

Reply:
My initial idea was to set a `env` var as a parameter named `MAX_REQUEST_TIMEOUT = 20 * 60 * 1000`  (20 mins in milliseconds) so that it can be set by the user.